<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColoredWaveView</class>
 <widget class="QWidget" name="ColoredWaveView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnEliminaOndaDue">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIndiceOndaUno">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>indice</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInserisciOndaDue">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widgetRisultato" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>521</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci valori onda</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci valori onda </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>indice</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widgetOndaDue" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>341</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSostituisciOndaUno">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sostituisci</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLunghezzaOndaDue">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lunghezza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInserisciOndaUno">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSommaOndeColorate">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOttieniOndaDue">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ottieni</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminaOndaUno">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDivisioneOndeColorate">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>711</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLunghezzaOndaUno">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditValoriOndaDueOndaSx">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Formato: numero, numero</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditValoriOndaUnoOndaSx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Formato: numero, numero</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widgetOndaUno" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMoltiplicazioneOndeColorate">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSottrazioneOndeColorate">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIndiceOndaDue">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOttieniOndaUno">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ottieni</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lunghezza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSostituisciOndaDue">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sostituisci</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>20</width>
     <height>681</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetColoriOndaUno">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>41</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>550</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>600</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>G:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>650</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditROndaUno">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>550</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGOndaUno">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>600</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBOndaUno">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>650</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderROndaUno">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>580</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderGOndaUno">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>630</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderBOndaUno">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>680</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStringOndaUno">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>570</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>550</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Stringa:</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderROndaDue">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>580</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderGOndaDue">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>630</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>650</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>550</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>550</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Stringa:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetColoriOndaDue">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>550</y>
     <width>41</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStringOndaDue">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>570</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGOndaDue">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>600</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderBOndaDue">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>680</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>600</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>G:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditROndaDue">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>550</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBOndaDue">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>650</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>views/utils/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
